### PR TITLE
Weather requests

### LINKED ISSUE #7: Serializer
- [x] Create serializer for weather service

### LINKED ISSUE #9: End to End Connection
- [x] BE engine is talking to microservice and fully functioning

### LINKED ISSUE #18: Weather Requests
- [x] Implement happy path request spec
- [x] Create weather service controller  method for `get /api/v1/forecast_data`
- [x] Require all necessary files in controller, POROS, facade and service files
- [x] Create a helper method for all Sinatra WeatherService App request specs to run the app

### PR DESCRIPTION
**Pull request by klatour324**
### What Changed?
- Add request spec happy path for forecast data
- Update serializer, poros, facade and controller files with correct file require configuration
- Fix service spec with correct coordinates for Chicago data
- Implement initial specs and code for sad path specs
### What's Next?
- BE to microservice connection
- Sad path/edge case testing
### Known Issues?
- Running into issues with error handling with Sinatra app and sad path specs
### Test Coverage?
- 100%
### README & Documentation
- Have endpoints documented in a gist. Need to get the README up to date with the necessary information

### CODE REVIEW

**Review by jmichlitsch — APPROVED** [review]
Good work completing the happy path for the forecast endpoint. We both just need to work more on the sad path testing for these and the micro service will be good to go

File: app/facades/weather_facade.rb [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+
 class WeatherFacade
   def self.forecast(location)
     location_data = MapQuestService.call(location)
     coordinates = lookup(location)
-    #weather_data = WeatherService.call(coordinates[:lat], coordinates[:lng])
-    #ForecastPoro.new(weather_data)
+    weather_data = WeatherService.get_forecast_for_location(coordinates[:lat], coordinates[:lng])
+    forecast = Forecast.new(weather_data)
   end
```
Comment (amsmyth1) [inline app/facades/weather_facade.rb:20]: Wow Facade pattern in Sinatra ~~~you are fancy~~~! great job!

**Review by amsmyth1 — APPROVED** [review]
This looks great! I am really impressed the work yall got done, specifically implementing a Facade Pattern!
